### DISCUSSION TITLE
Problems when viewing .ico files in the mobile app

### DISCUSSION THREAD

**Discussion by michelelorusso05** [community/community#47452] [Mobile]
It seems that while loading .ico files the app gets stuck in some sort of infinite loop that continuously allocates resources, leading to a massive slowdown of the device and unresponsiveness of the app, without the opened file being actually showed.

For reference, I'm trying to open the cpp.ico file in the Microsoft/vscode repository under the resources folder.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @michelelorusso05 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by sami1250** [reply]
gh repo clone TwilioDevEd/api-snippets

**Comment by sami1250** [comment]
gh repo clone TwilioDevEd/api-snippets
